langchain4j-workers-ai/src/main/java/dev/langchain4j/model/workersai/WorkersAiModelName.java

```diff
@@ -0,0 +1,218 @@
+package dev.langchain4j.model.workersai;
+
+/**
+ * Group model name in a class to ease the usage.
+ * It has been preferred as an Enum for flexibility.
+ */
+@SuppressWarnings("unused")
+public class WorkersAiModelName {
```

Comment:
minor: would be nice to have separate enums for each model type, similar to `OpenAiChatModelName`/`OpenAiEmbeddingModelName`/etc